draft-kowalik-rpp-architecture-02.adoc

```diff
@@ -177,7 +177,7 @@ It encompasses aspects such as:
 * **Resource type definitions:** Defining the structure of specific resource types by combining data elements.
 * **IANA registry definitions:** Potentially registering resource definitions with IANA for standardized and automated processing.
 * **Mapping of data elements to media types:** Defining how the data elements of a resource type are represented in different media types (e.g., JSON, XML).
-* **Extensibility mechanisms on the resource type level:** Providing mechanisms for extending resource types with new data elements or operations.
+* **Extension mechanisms:** Providing mechanisms for creating new resource types and for extending existing resource types with new data elements or operations.

```

Comment:
I think it does not yet fully address #38.
... or operations including potentially new response status codes.